src/NuGet.Clients/NuGet.PackageManagement.UI/Properties/AssemblyInfo.cs

```diff
@@ -24,4 +23,4 @@
 
 [assembly: ThemeInfo(ResourceDictionaryLocation.None, ResourceDictionaryLocation.SourceAssembly)]
 [assembly: XmlnsPrefix("http://schemas.nuget.org/xaml", "nuget")]
-[assembly: XmlnsDefinition("http://schemas.nuget.org/xaml", "NuGet.PackageManagement.UI")]
\ No newline at end of file
+[assembly: XmlnsDefinition("http://schemas.nuget.org/xaml", "NuGet.PackageManagement.UI")]
```

Comment:
The whole file seems like a non-functional change, can you revert it maybe?

Reply:
yes, will do. i added it...then removed it based on andy's feedback.  will revert file.